spec/integration/provider/acl/windows_spec.rb

```diff
@@ -573,14 +573,16 @@ def get_permissions_for_path(path)
           end
           resource[:permissions] = permissions_hash
 
-          set_perms(permissions).must == permissions
+          
         end
 
         it 'removes the permissions successfully' do
+          expect(set_perms(permissions)).to eq permissions
```

Comment:
It looks like this test was moved from the previous it block, which leaves it without any tests, just set-up?

Reply:
@michaeltlombardi  `expects` are not allowed in `before` blocks due to rubocop so I moved the `expect` into the actual test.